src/Google/Ads/GoogleAds/Lib/V14/GoogleAdsClientBuilder.php

```diff
@@ -415,6 +416,48 @@ public function validate()
                 'The gRPC channel credential can only be set when the transport is "grpc".'
             );
         }
+        // Check if the version of the grpc extension installed by Composer is greater than that of
+        // the extension installed by PECL, throw an exception to remind the user to upgrade.
+        $grpcPackageVersion = floatval(phpversion('grpc'));
+        $grpcComposerVersion = $this->getGrpcComposerVersion();
+        if (!empty($grpcComposerVersion) &&  $grpcComposerVersion > $grpcPackageVersion) {
+            throw new UnexpectedValueException(
+                'The grpc extension installed by Composer has a greater version than that'
+                . ' installed by PECL. Upgrade the PECL extension to avoid issues caused by the'
+                . ' version difference. For linux, run "sudo pecl install grpc".'
+            );
+        }
+    }
+
+    /**
+     * Gets the grpc version installed by Composer from the composer.lock file. Returns null if the
```

Comment:
`Returns null if the if that information` ->
`Returns null if that information`